### ISSUE TITLE
Error when installing Autoraise 4.1 on MacOS Sonoma

### ISSUE THREAD

**Issue by IAmVisco** [Dimentium/homebrew-autoraise#8]
```
==> make clean
==> make
Last 15 lines from /Users/user/Library/Logs/Homebrew/autoraise/02.make:
                               ^
AutoRaise.mm:1046:37: error: use of undeclared identifier 'is_chrome_app'
                                    is_chrome_app(frontmostApp.bundleIdentifier));
                                    ^
AutoRaise.mm:1048:87: error: use of undeclared identifier 'focusedWindow_psn'
                                    OSStatus error = GetProcessForPID(frontmost_pid, &focusedWindow_psn);
                                                                                      ^
AutoRaise.mm:1049:51: error: use of undeclared identifier '_focusedWindow_psn'
                                    if (!error) { _focusedWindow_psn = &focusedWindow_psn; }
                                                  ^
AutoRaise.mm:1049:73: error: use of undeclared identifier 'focusedWindow_psn'
                                    if (!error) { _focusedWindow_psn = &focusedWindow_psn; }
                                                                        ^
6 errors generated.
make: *** [AutoRaise] Error 1
```

**Comment by jmechnich** [comment]
I can reproduce the error when doing a fresh `brew install autoraise` without using any flags.
For now you can get it to work adding `--with-dexperimental_focus_first`.

**Comment by sbmpost** [comment]
@IAmVisco @jmechnich 

This is fixed in v4.2. thanks for reporting
